packages/common/src/scopeSupportFacets/r.ts

```diff
@@ -0,0 +1,189 @@
+import type { LanguageScopeSupportFacetMap } from "./scopeSupportFacets.types";
+import { ScopeSupportFacetLevel } from "./scopeSupportFacets.types";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const { supported, unsupported, notApplicable } = ScopeSupportFacetLevel;
+
+export const rScopeSupport: LanguageScopeSupportFacetMap = {
```

Comment:
Could you please group these in supported/unsupported/notApplicable? Have a look at how we have done in other languages.